avm/res/app/container-app/tests/e2e/vnet/main.test.bicep

```diff
@@ -0,0 +1,73 @@
+targetScope = 'subscription'
+
+metadata name = 'VNet integrated container app deployment'
+metadata description = 'This instance deploys the container app in a managed environment with a virtual network using TCP ingress.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-app.containerApps-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'acavnet'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+// =========== //
+// Deployments //
+// =========== //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2023-07-01' = {
+  name: resourceGroupName
+  location: resourceLocation
+}
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-paramNested'
+  params: {
+    location: resourceLocation
+    managedEnvironmentName: 'dep-${namePrefix}-menv-${serviceShort}'
+  }
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+@batchSize(1)
+module testDeployment '../../../main.bicep' = [
+  for iteration in ['init', 'idem']: {
+    scope: resourceGroup
+    name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}-${iteration}'
+    params: {
+      name: '${namePrefix}${serviceShort}001'
+      environmentId: nestedDependencies.outputs.managedEnvironmentResourceId
```

Comment:
Hm, this parameter should actually be `environmentResourceId` 😄

Reply:
@AlexanderSehr I've updated the param. I just bumped the minor version as wasn't sure if I was supposed to go to 1.0.0 yet?

Reply:
Right, while we're not on 1.0, we can only effectively bumb minor. Thanks 🚀